client/src/partials/fiscal/fiscal.openingBalance.html

```diff
@@ -0,0 +1,88 @@
+<style media="screen">
```

Comment:
Can you put these into the `structure.css` file?  It's bad practice to include style tags in `<html>` since they cannot be scoped.  See [the MDN article](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope) on scoped CSS to show an experimental solution to this problem.

Note that the MDN article does not work in Chrome.